mev-boost-rs/src/relay_mux.rs

```diff
@@ -124,18 +126,40 @@ impl BlindedBlockProvider for RelayMux {
         &self,
         bid_request: &BidRequest,
     ) -> Result<SignedBuilderBid, BlindedBlockProviderError> {
-        let responses = stream::iter(self.relays.iter().cloned())
-            .map(|relay| async move { relay.fetch_best_bid(bid_request).await })
+        let try_responses = stream::iter(self.relays.iter().cloned())
+            .enumerate()
+            .map(|(relay_index, relay)| async move {
+                let bid = tokio::time::timeout(
+                    std::time::Duration::from_secs(FETCH_BEST_BID_TIME_OUT as u64),
```

Comment:
you want to pull in `Duration` via an import `use std::time::Duration;` ?

imo a bit more idiomatic